internal/provider/waypoint/data_source_waypoint_application_template_test.go

```diff
@@ -52,19 +48,61 @@ func TestAccWaypointData_Application_Template_basic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceName, "name", updatedName),
 					resource.TestCheckResourceAttr(dataSourceName, "name", updatedName),
-					resource.TestCheckResourceAttr(dataSourceName, "variable_options.0.name", "string_variable"),
-					resource.TestCheckResourceAttr(dataSourceName, "variable_options.0.variable_type", "string"),
-					resource.TestCheckResourceAttr(dataSourceName, "variable_options.0.options.#", "1"),
-					resource.TestCheckResourceAttr(dataSourceName, "variable_options.0.options.0", "a"),
```

Comment:
why are these checks being removed?

Reply:
This test for the data source used the "basic" acceptance test resource config. That resource's template has a no-code module which I updated in a previous PR to be v0.0.3, which included variable options (to test that feature). We had a discussion where you requested that be tested _not_ in the "basic" acceptance test, so I [rolled that template's no-code module version back to v0.0.2 ](https://github.com/hashicorp/terraform-provider-hcp/pull/833/files#diff-24fe8b2f639120d51e8271a8582eac54f7c2c76be48da056f541f917559b38a2R162)(with _no_ variable options). That precipitated this change, because now there are no variables on the template, or the application that used that template.